### PR TITLE
start loading messages earlier

### PR DESCRIPTION
**Pull request by flexsurfer**
start loading messages earlier

### CODE REVIEW

File: src/status_im/chat/models.cljs [hunk 1]
```diff
@@ -295,16 +245,17 @@
               (when-not (group-chat? cofx chat-id)
                 (transport.filters/load-chat chat-id))
               (when platform/desktop?
-                (mark-messages-seen chat-id))
+                (message-seen/mark-messages-seen chat-id))
               (when (and (one-to-one-chat? cofx chat-id) (not (contact.db/contact-exists? db chat-id)))
-                (contact.core/create-contact chat-id)))))
+                (contact.core/create-contact chat-id))
+              (loading/load-messages))))
```
Comment (flexsurfer) [inline src/status_im/chat/models.cljs:251]: only one change is here, all other because of cycle deps :(

**Comment by flexsurfer** [comment]
@cammellos tested  on real device, i can't see chat logo anymore, so much better

File: src/status_im/chat/models.cljs [hunk 2]
```diff
@@ -295,16 +245,17 @@
...
 
 (fx/defn navigate-to-chat
   "Takes coeffects map and chat-id, returns effects necessary for navigation and preloading data"
   [cofx chat-id]
   (fx/merge cofx
-            (navigation/navigate-to-cofx :chat {})
-            (preload-chat-data chat-id)))
+            (preload-chat-data chat-id)
```
Comment (cammellos) [inline src/status_im/chat/models.cljs:257]: Is the order important here? Maybe we should leave a comment if so.

**Review by cammellos — APPROVED** [review]
@flexsurfer nice!

File: src/status_im/chat/models.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im/chat/models.cljs:257]: re-frame doesn't guaranty order of fx handlers, so I did it just in case :)

**Pull request merged**